ext/pxp-agent.logrotate

```diff
@@ -0,0 +1,11 @@
+/var/log/puppetlabs/pxp-agent/*.log {
+    daily
+    missingok
+    rotate 30
+    compress
```

Comment:
It looks like logrotate compresses (with gzip) by default, but I have added that explicitly here.